fxa/requests.py

```diff
@@ -10,7 +10,10 @@
 from requests.auth import AuthBase
 from six.moves.urllib.parse import urlparse
 
-from fxa.core import DEFAULT_SERVER_URL, Client
+from fxa import core
+from fxa import oauth
+
+DEFAULT_CLIENT_ID = "5882386c6d801776"  # Firefox dev Client ID
```

Comment:
The idea is not to bother the user with it so they can get started pretty fast. Also they can configure it using the ``FXA_CLIENT_ID`` environment variable.

I used this ID because it was already present in all Firefox Account deployments so that the user doesn't bother about it. If we can add a PyFxA one that would be great I think.